app/services/transaction.service.js

```diff
@@ -148,6 +148,15 @@ function getStatusCodeForError (err, response) {
   return status
 }
 
+function handleErrorForFailedSearch (err, response) {
+  const code = (response || {}).statusCode || (err || {}).errorCode
+  if (code === 504) {
+    return new GatewayTimeoutError('Your request has timed out. Please apply more filters and try again.')
```

Comment:
I assume this wording came from or has been approved by a content designer. Could you confirm that for the record, please?

Reply:
Yes by Alan Maddrell and Nath Driver.